tests/unit/file_export/test_transcribe_exceptions.py

```
@@ -0,0 +1,101 @@
+        mock_audio.dBFS = -10  # above silence threshold
+        mock_audio.export = MagicMock()
+
+        mock_segment = MagicMock()
+        mock_segment.dBFS = -10
+        mock_audio.__getitem__.return_value = mock_segment
+        mock_segment.export = MagicMock()
+
+        with patch("fastfetchbot_file_export.transcribe.OpenAI", return_value=mock_client), \
+             patch("fastfetchbot_file_export.transcribe.AudioSegment") as MockAudioSegment, \
+             patch("fastfetchbot_file_export.transcribe.os.remove") as mock_remove, \
+             patch("fastfetchbot_file_export.transcribe.os.path.splitext", return_value=("/tmp/audio", ".mp3")), \
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/audio"

(S108)